### PR TITLE
fix(metrics): revert metric names to pre-Java

### PR DESCRIPTION
**Pull request by mattgogerly**
closing spinnaker/spinnaker#6821

### CODE REVIEW

**Comment by dbyron-sf** [comment]
FWIW, looks like [this](https://github.com/spinnaker/clouddriver/pull/5703/files#diff-f700e923bebcf36bf8779629c6fe081c40b0b58417d07e0a9a2da04344918bafR46) introduced the bug.  If the PR labels are right, it first went in to 1.29.

File: clouddriver-core/src/test/java/com/netflix/spinnaker/clouddriver/cache/MetricInstrumentationTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import com.netflix.spinnaker.cats.agent.CachingAgent;
+import com.netflix.spinnaker.cats.provider.ProviderCache;
+import java.util.Collection;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
```
Comment (dbyron-sf) [inline clouddriver-core/src/test/java/com/netflix/spinnaker/clouddriver/cache/MetricInstrumentationTest.java:38]: I'm not seeing any `@Mock` annotations, so do we need this?  I took it out (and the two imports) and the test still passes for me.

Comment (mattgogerly) [reply clouddriver-core/src/test/java/com/netflix/spinnaker/clouddriver/cache/MetricInstrumentationTest.java:38]: Nope not required, leftover from before I changed the test.

**Comment by dbyron-sf** [comment]
LGTM.  Let's wait until https://github.com/spinnaker/clouddriver/pull/5938 is in before going ahead with this.

The joy of the monorepo.  I can't wait.

**Review by dbyron-sf — APPROVED** [review]

**Comment by mattgogerly** [comment]
@Mergifyio backport release-1.29.x release-1.30.x

**Pull request merged**